src/main/java/com/bandwidth/sqs/consumer/strategy/expiration/VisibilityTimeoutPercentageExpiration.java

```diff
@@ -0,0 +1,23 @@
+package com.bandwidth.sqs.consumer.strategy.expiration;
+
+import com.bandwidth.sqs.queue.SqsMessage;
+
+import java.time.Duration;
+
+/**
+ * Considers a message expired after it's age is a certain percentage of the visibility timeout
```

Comment:
"it's" -> "its"

Reply:
Bikeshedding Benson

Reply:
I don't know if that fits the definition of bikeshedding